src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,43 @@
+package duke.command;
+
+import java.io.IOException;
+
+import duke.task.Task;
+import duke.utility.Storage;
+import duke.utility.TaskList;
+import duke.utility.Ui;
+
+
+/**
+ * Represents a command to add a {@link Task} in a {@link TaskList}.
+ */
+public class AddCommand extends Command {
```

Comment:
very well organised! I like how you divided the commands into different classes which made the overall code more readable :)